MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp

```diff
@@ -1261,6 +1300,163 @@ void EnggDiffractionViewQtGUI::setListWidgetBank(int idx) {
   selectBank->setCurrentRow(idx);
 }
 
+void MantidQt::CustomInterfaces::EnggDiffractionViewQtGUI::
+    fittingRunNoChanged() {
+  try {
+    QString focusedFile = m_uiTabFitting.lineEdit_pushButton_run_num->text();
+    std::string strFocusedFile = focusedFile.toStdString();
+    // file name
+    Poco::Path selectedfPath(strFocusedFile);
+    Poco::Path bankDir;
+
+    // handling of vectors
+    m_fitting_runno_dir_vec.clear();
+    std::string strFPath = selectedfPath.toString();
+    std::vector<std::string> splitBaseName = splitFittingDirectory(strFPath);
+
+    if (selectedfPath.isFile() && !splitBaseName.empty()) {
+
+#ifdef __unix__
+      auto home = Poco::Path().home();
+      bankDir.append(home);
+      bankDir.append(selectedfPath.parent().toString());
+      bankDir.append(selectedfPath);
+#else
```

Comment:
@FedeMPouzols The only thing I can think of here is that for linux, I may not require to append `home` as well as  `selectfPath.parent()` .. otherwise it should work as we are utilising the same technique [here](https://github.com/mantidproject/mantid/blob/master/MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionPresenter.cpp#L2952-L2957) too